package.json

```diff
@@ -119,11 +119,13 @@
     "@types/vfile-message": "^2.0.0",
     "@typescript-eslint/eslint-plugin": "^4.9.0",
     "@typescript-eslint/parser": "^4.9.0",
+    "autoprefixer": "^9.8.6",
```

Comment:
Dunno if it seems like a better option to you, but postcss-preset-env would bundle up some of these features into a set https://github.com/csstools/postcss-preset-env